src/serac/infrastructure/output.cpp

```diff
@@ -68,7 +70,9 @@ void outputFields(const axom::sidre::DataStore& datastore, const std::string& da
   conduit::Node extracts;
   // "relay" is the Ascents Extract type for saving data
   extracts["e1/type"]            = "relay";
-  extracts["e1/params/path"]     = fmt::format("{}_fields.{}", data_collection_name, file_format_string);
+  const std::string file_name    = fmt::format("{}_fields.{}", data_collection_name, file_format_string);
```

Comment:
Now that we're saving histories directly to sidre, are these ascent extracts still useful?

Reply:
I'll start with turning them off by default.  Then revisit this in a follow up pr.